packages/f/fontsource-abeezee.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "fontsource-abeezee",
+  "description": "Self-host the ABeeZee font in a neatly bundled NPM package.",
+  "main": "index.css",
```

```suggestion
  "filename": "index.css",
```
The recommended default file for the library in the UI is indicated with `filename`, not `main`